facia/app/utils/DotcomFrontsLogger.scala

```diff
@@ -27,7 +27,7 @@ case class DotcomFrontsLogger() extends GuLogging {
       ),
       LogFieldString(
         "front.collections",
-        faciaPage.collections.map(_.collectionType).mkString(", "),
+        JSON.stringify(faciaPage.collections.map(_.collectionType)),
```

Comment:
Might need an import (can't do it my end, laptop froze).

Reply:
```suggestion
        Json.stringify(Json.toJson(faciaPage.collections.map(_.collectionType))),
```

Reply:
Thanks